homeassistant/components/alarm_control_panel/alarmdecoder.py

```diff
@@ -96,3 +96,9 @@ def alarm_arm_home(self, code=None):
         if code:
             _LOGGER.debug("alarm_arm_home: sending %s3", str(code))
             self.hass.data[DATA_AD].send("{!s}3".format(code))
+
+    def alarm_arm_custom_bypass(self, code=None):
+        """Send custom bypass command."""
+        if code:
+            _LOGGER.debug("alarm_arm_custom_bypass: sending %s", str(code))
```

Comment:
I don't think we should expose the code in the log.

Reply:
Check the full file, all defined method do this exact logging.